dev/vbox/setup

```diff
@@ -14,9 +14,6 @@ if VBoxManage list hostonlyifs | grep vboxnet0 >/dev/null; then
     abort "'vboxnet0' interface has DHCP enabled; please disable it"
   fi
 
-  if ! VBoxManage list hostonlyifs | grep -A3 vboxnet0 | grep 'IPAddress:\s\+192.168.50.1' >/dev/null; then
```

Comment:
heey, was this removal intended? wondering if that's leftovers from local changes :grin:

Reply:
Oh, ya... That was just to get us unblocked locally when testing. Feel free to revert that change.

Reply:
force pushed a fix to that commit @cirocosta